Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.11-slim-buster
+FROM python:slim
+
+LABEL org.opencontainers.image.source="https://github.com/toddrob99/searcharr"
+LABEL org.opencontainers.image.description="Docker for SEARCHARR"
+LABEL Name=Searcharr Version="v1.2-beta"
+
+ARG TARGETPLATFORM BUILDPLATFORM
```

Comment:
What is the purpose of these? I don't see the args being used anywhere.

Reply:
```ARG TARGETPLATFORM BUILDPLATFORM``` is a must for multi architecture builds

and using labels are not necessary but for tagging name and dockerhub repository to this github repository it is used...

Reply:
Are the ARGs what was stopping the arm64/v8 arch build from working? Why do the other archs work without these ARGs?

Reply:
Please also remove `-beta` from the Dockerfile version and increment to `v1.3`